hilo.rb

```diff
@@ -1,2 +1,20 @@
-number = rand(1..100)
-puts number 
\ No newline at end of file
+random_number = rand(1..100)
+puts "Enter a number between 1 and 100, then press enter to submit your guess. You have 6 chances to guess the correct number"
+input = gets.chomp
+guess = input.to_i
+count = 0
+while count < 5
+    count = count + 1
+    if guess.to_s != input
+        puts "Your guess is not a whole number. Try again"
+    elsif guess > 100
+        puts "Your guess is too high. Try again"
+    elsif guess < 1
+        puts "Your guess is too low. Try again"
+    elsif guess == random_number
```

Comment:
I think with the change suggested above you wouldn't need an `eslif` here.

Reply:
Ah that makes everything make a lot more sense lol, now I see that if the number is not higher or lower than the random number it has to be correct so the check is no longer required here.